### DISCUSSION TITLE
[FE-Container] Header removed

### DISCUSSION THREAD

**Discussion by valesceva** [mia-platform/community#104] [Q&A] [answered]
Hi, i'm trying to add header X-Frame-Options: DENY to website.conf in the nginx configuration of fe-container, the header is setted by the micro-service: if i try calling it directly through port-forward. But when I start the application in the environment the header is no longer present, could it be API-GATEWAY fault ? And if it is, how can i fix the problem?

**Comment by davidebianchi** [comment, marked as answer]
Hi @valesceva! What api-gateway are you using?

**Reply by valesceva** [reply]
Hi, we are using version 4.1.4

**Reply by theliuk** [reply]
nginx, not envoy

**Reply by davidebianchi** [reply]
I add a test on the api-gateway based on nginx, and it seems to correctly return the header so it is a strange behaviour. Can you give me more info about your configuration? Have you some advanced configuration on api-gateway?

**Reply by theliuk** [reply]
Hi, we don't have any particular configuration. If you had ten minutes we could show you.

Thanks,

Luca

**Reply by davidebianchi** [reply]
We saw together. There are not issues with api-gateway, headers it's correctly forwarded but there are multiple exposed website and only one exposes it.
In the new version of the Console it will be set by default to SAMEORIGIN as security standard (if not passed from the upstream). Or, if you want to use DENY or you can't await the next Console version, you should set it in every website.

**Reply by valesceva** [reply]
Adding header to all services fixed the problem, thanks
